DS-2007: add batchSize commandline option to bin/dspace cleanup;  make it continue past exceptions

akinom opened the pull request:
add option to control commit batch size to    bin/dspace cleanup 

cleaned BitstreamStorageManager code up a bit while moving exception handling inside the loop that goes over deleted bitstreams 

https://jira.duraspace.org/browse/DS-2007

pnbecker committed:
DS-2490: Add versioned DOIs and adapt DOI support to service API

pnbecker committed:
DS-1349: Add configuration to hide submitter in item history (JSPUI).

pnbecker committed:
DS-2547: Adds a label to access restricted bitstreams in JSPUI.

This feature is configurable (using configuration property
webui.itemdisplay.label.restricted.bitstreams).

tdonohue committed:
Initial refactor commit. Swap out DSpaceConfig/ServiceConfig for Apache Commons Configuration

tdonohue committed:
Remove completely unused constants and configurations. These are not used at all.

tdonohue committed:
Add back in a few dynamically generated configurations which were previously removed during refactor

tdonohue committed:
Fix loading of config-definition.xml in Unit Testing. Look for it on ClassPath if cannot be found elsewhere.

tdonohue committed:
Copy major convenience methods from ConfigurationManager to ConfigurationService. Add new Unit Tests to verify.

tdonohue committed:
Add a few more unit tests of reading properties of different types. Auto-trim all string values

tdonohue committed:
Rewrite ConfigurationManager to wrap ConfigurationService. Removal of "dspace-config" and "dspace.configuration" system/servlet params

tdonohue committed:
More convienence methods to ConfigurationService. "hasProperty()" and "getArrayProperty()". Includes unit tests.

tdonohue committed:
Minor services tweaks. Enhance comments. Have SpringServiceManager use getArrayProperty() instead of getPropertyAsType()

tdonohue committed:
Refactor org.dspace.core.* classes to use new ConfigurationService. Helps pass more unit tests.

tdonohue committed:
Prepend all configs with module name (to ensure unique keys). Load all module configs from dspace.cfg via "include" statements.

tdonohue committed:
Update multivalued properties to use getArrayProperty from ConfigurationService. Update specific config names based on renamed config keys.

tdonohue committed:
Fix compilation error after rebase

PhilipVis committed:
DS-2880: Pubmed integration into XMLUI submission

tdonohue committed:
Refactor PluginManager into PluginService and LegacyPluginServiceImpl. Add unit tests.

tdonohue committed:
Refactor ChoiceAuthorityServiceImpl to load necessary plugins on first use, NOT on initialization

tdonohue committed:
Enhance error reporting

tdonohue committed:
Refactor several beans (and related classes) to use ConfigurationService on init once it is injected.

tdonohue committed:
Create new AbstractDSpaceTest to initialize Kernel separate from Database. Some tests ONLY need the Kernel for configs, and not the full DB.

tdonohue committed:
Test needs access to Configs / Kernel

tdonohue committed:
Ensure dspace.dir system property also specified for Integration Tests

tdonohue committed:
Replace MockConfigurationManager with JMockit Expectations statements

PhilipVis committed:
29313: import citations framework integration into dspace6

29313: Integrate import citations framework into dspace6

29412: import external package refactoring

29412: import external package refactoring

more refactorying

pnbecker committed:
DS-2497: Keep version numbers stable.

tdonohue committed:
Allow ScriptLauncher to run *without* ConfigurationManager. Requires tweaking how DSpaceConfigurationService 'discovers' the DSpace home directory (new unit tests added).

tdonohue committed:
Fix JSP reference to old PluginManager. Update comments referencing PluginManager. Remove old plugin.reusable.* configs as they are no longer needed

tdonohue committed:
Implement dynamic property substitutions for our Spring Config files.

tdonohue committed:
Stop filtering properties in all POMs. With Commons Config, we'll load them dynamically

tdonohue committed:
Remove build.properties and replace it with a "local.cfg" which can override configs as needed.

tdonohue committed:
Update Hibernate configuration to load dynamic settings via Spring / ConfigurationService. Add default DB settings back to dspace.cfg

tdonohue committed:
Update Test Environment to use its own local.cfg INSTEAD OF dspace.cfg.more / merging multiple configs

tdonohue committed:
Correct missing license headers

PhilipVis committed:
29600: importer framework readme

PhilipVis committed:
29600: importer framework readme

PhilipVis committed:
29600: importer framework readme

PhilipVis committed:
29600: Document framework

PhilipVis committed:
29600: Document framework

Commit:
further ameliorations to the readme

Commit:
readme bte explanation

tdonohue committed:
Stop filtering configs during Maven assembly or Ant build

tdonohue committed:
Add back in log4j initialization via a new LoggerServiceImpl which loads configs from Commons Configs

tdonohue committed:
Ignore local.cfg by default. Provide an EXAMPLE. Make sure Maven assembly handles local.cfg. Make sure Ant build reads local.cfg

tdonohue committed:
Correct build.xml, make it backwards compatible with old versions

tdonohue committed:
Remove more filtering from POMs. No longer used.

tdonohue committed:
Remove unnecessary filtering

Commit:
readme ameliorations

Commit:
fix link in readme

jonas-atmire committed:
Addition of license headers

tdonohue committed:
Configuration cleanup. Restore authentication.cfg. Add example of configuring AuthN to local.cfg. Move default logging settings to dspace.cfg. Uncomment all module configs

tdonohue committed:
Fix comment

pnbecker committed:
DataCiteConnector: don't rely on handles.

While the DOIIdentifierProvider checks our database to ensure that a DOI
is not reserverd/registered for another DSpaceObject.
The DataCiteConnector was checking this again agains  the meta data we
send to DataCite. Within these meta data we are submitting the handle as
alternative identifier and used it to check whether a DOI is
reserved/registered for a specific DSpaceObject. This commit removes
the checks from the DataCiteConnector for the following reasons:
 * In case we want to completly replace handles by DOIs one day, we
   shouldn't rely on handles. Further ongoing: an IdentifierProvider
   shouldn't rely on other identifiers than the one he provides.
 * If Item Level Versioning is enabled the handles may change depending
   on the VersionedHandleIdentifier that is used. This makes handles
   unreliable for the checks we were using them before.
   handles.

pnbecker committed:
DOIConumser: Ignore Workspace and Workflow items, reduce logging

pnbecker committed:
IdentifierServiceImpl.reserve(ctx, dso) should mint and reserve

KevinVdV committed:
Create factory to retrieve an instance of the "configurationService" object

tdonohue committed:
Minor bug fixes. Move bean definition to -services and ensure proper imports

rradillen committed:
mirage 2 not as default theme

KevinVdV committed:
Allow the local cfg to be filtered by maven

tdonohue committed:
Re-enable web.xml filtering in Maven POMs. It's necessary for running/debugging webapps in IDEs

tdonohue committed:
Reenable Ant filtering of specific configs which cannot utilize Commons Config. Also fix/cleanup Solr's log4j configuration.

jonas-atmire committed:
DS-2880 Pubmed docs + Mirage 1 integration

jonas-atmire committed:
DS-2880 Licence header exclusion + lowercase change

tdonohue committed:
Refactor REST API and code from PR #1239 to use ConfigurationService and PluginService. Rename rest.cfg configs to use proper prefixing.

tdonohue committed:
Found an overlooked configuration needing renaming. "dspace.oai.url" is now simply "oai.url" in oai.cfg. Fixed references elsewhere & updated them to use ConfigurationService

tdonohue committed:
Update code changes from PR #970 to utilize PluginService and ConfigurationService

helix84 committed:
Merge pull request #1245 from AmberPoo1/correct-conformsTo-dcterm

Correct dcterm "dcterms.conformsTo" in registry configuration

jonas-atmire committed:
DS-2880 Usage of localstorage to store compiled templates

tamu-sad-iii committed:
Added CSVExportDiscoveryNavigation to apply tranform when pattern matching discovery.

tamu-sad-iii committed:
Fixed search metadata export by passing query and filter to processQuery of AbstractSearch.

tamu-sad-iii committed:
Removed some dead code.

tdonohue committed:
Minor bug fixes. Ensure controlpanel settings are loaded property (PR #970), and fix 'dspace dsprop' command which is useful in testing.

tamu-sad-iii committed:
Using query string instead of path. Some code improvement.

mwoodiupui committed:
[DS-1837] Copy the JAR to the installer 'lib' directory.

tdonohue committed:
Bug fix from previous commit. Accidentally added back in a non-existing config file

KevinVdV committed:
[DS-3003] Ensure all service & DAO implementations have a protected constructor

KevinVdV committed:
[DS-3003] Ensure all service & DAO implementations have a protected constructor

kosarko committed:
moving queries to DAOs

christian-scheible committed:
[DS-1518] Support of StartTLS in LDAPAuthentication.

tdonohue committed:
DS-3005: Ensure ItemService is initialized in all constructors

helix84 committed:
Merge pull request #1249 from tdonohue/DS-3005

DS-3005: Ensure ItemService is initialized in all constructors

tdonohue committed:
Merge pull request #801 from TAMULib/5x_search_result_export_metadata

DS-1262 CSV export of search results in XMLUI

tdonohue committed:
Rebase again, and add explanatory comments for new DSPR#801 config

tdonohue committed:
Remove two more deprecated XMLUI Lucene classes (accidentally overlooked and unused)

tdonohue committed:
Remove ALL lucene references from JARs. (NOTE: Lucene will still get pulled into DSpace though via Solr)

tdonohue committed:
Merge pull request #1235 from tdonohue/DS-2187-remove-lucene

DS-2187 : Remove Lucene search support

jonas-atmire committed:
DS-2880 "Reverting" localstorage + added additional check on DSpace var definition

KevinVdV committed:
Merge pull request #1244 from cjuergen/DS-2990

Fix for DS-2990

tdonohue committed:
Remove remaining DBMS Browse related classes (IndexBrowse, etc). Remove BrowseCreateDAO as it is not needed for Solr support. Refactor ItemCounter to remove methods which are no longer necessary for Solr support.

tdonohue committed:
Remove obsolete DBMS browse configurations (including Spring ones). Update/clarify comments for other browse related configs

tdonohue committed:
Remove duplicative RecentSubmissions logic (artifact of DBMS queries) from XMLUI

kosarko committed:
generics

kosarko committed:
report all exceptions from EmbargoCheck

kosarko committed:
healthcheck command email template

mwoodiupui committed:
Merge pull request #1160 from atmire/DS-2876

DS-2876 Framework for importing external metadata

mwoodiupui committed:
Merge pull request #1163 from atmire/DS-2880

DS-2880: Pubmed integration into XMLUI submission

helix84 committed:
Merge pull request #947 from saiful-semantic/master

DS-1187 Full-text indexing of right-to-left PDF files

tdonohue committed:
Minor code cleanup. MigrationUtils should be moved and all its methods made protected (as they bypass Hibernate). Update description of 'repair' command in DatabaseUtils.

tdonohue committed:
Flyway Java migration to remove all DBMS browse related tables, sequences and views

mwoodiupui committed:
[DS-1837] Pull remote Handle resolver out of DSpace and into a separate project

KevinVdV committed:
[DS-2992] Selection list for metadata in edit item or item template is not sorted alphabetically anymore

Commit:
Merge pull request #994 from ufal/DS-2659

DS-2659 new extensible platform health check reports with emailing

KevinVdV committed:
[DS-3008] Fix metadata order storage

KevinVdV committed:
Merge pull request #1221 from atmire/DS-2945

DS-2945 Submitting to a collection without any workflow as normal user ends in an Exception

mwoodiupui committed:
Merge pull request #1104 from tdonohue/DS-2654-common-config

DS-2654: Enhanced Configurations via Apache Commons Configuration

mwoodiupui committed:
[DS-3010] Fix a missed use of PluginManager

mwoodiupui committed:
[DS-3010] Correct configuration names.

mwoodiupui committed:
Merge pull request #1258 from mwoodiupui/DS-3010

[DS-3010] integrating DS-2654 uncovered some missed PluginManager conversion

helix84 committed:
Merge pull request #848 from christian-scheible/DS-2426

[DS-2426] added possibility to use relative import in oai xslt transformations

helix84 committed:
Merge pull request #1256 from KevinVdV/DS-3008-fix-metadata-order

[DS-3008] Fix metadata order storage

helix84 committed:
Merge pull request #1255 from KevinVdV/DS-2992-metadatafield-retrieval-order

[DS-2992] Selection list for metadata in edit item or item template is not sorted alphabetically anymore

helix84 committed:
Merge pull request #1240 from KevinVdV/DS-2989-fix-workspaceitem-view

[DS-2989] View workspace item broken in JSPUI

tim-atmire committed:
Quick hotfix for the Collection roles render

tim-atmire committed:
Merge pull request #2 from tim-atmire/XMLUI-AssignCollectionRoles-hotfix

Xmlui assign collection roles hotfix

tdonohue committed:
Merge pull request #1259 from tim-atmire/master

DS-3015 Collection roles page needs updating

helix84 committed:
DS-3016 revert failing upgrades: ehcache xerces httpclient junit h2

helix84 committed:
DS-3014 LDAP authentication not working

pnbecker committed:
DS-2879/XmlWorkflow: keep the old behavior as default

This PR harmonizes the default behavior of the old workflow system and
XMLWorkflow system regarding whether notifications for tasks returned
to the task pool should be send or not.

pnbecker committed:
Merge pull request #1262 from tuub/DS-2879-XML

DS-2879/XmlWorkflow: keep the old behavior as default

helix84 committed:
Merge pull request #1105 from KevinVdV/DS-2807-fix-bundle2bitstream-order

[Task: DS-2807] Bundle bitstream ordering fix for hibernate

tdonohue committed:
Merge pull request #1261 from helix84/DS-3014-escape-commas-in-config

DS-3014 LDAP authentication not working

peterdietz committed:
Merge branch 'DS-2784-jersey2' of https://github.com/akotynski/DSpace into akotynski-DS-2784-jersey2

Conflicts:
	dspace-rest/src/main/java/org/dspace/rest/RestReports.java

peterdietz committed:
Merge branch 'akotynski-DS-2784-jersey2'

terrywbrady committed:
remove unneeded decl

helix84 committed:
DS-3016 upgrade dependencies before 6.0 release

helix84 committed:
Merge pull request #1263 from Georgetown-University-Libraries/ds-3018

DS-3018 Remove unneeded declaration causing exception in /rest/reports

mwoodiupui committed:
[DS-3024] Add a 'permanent' attribute to Groups.

mwoodiupui committed:
[DS-3024] Add a 'permanent' attribute to Groups.

terrywbrady committed:
handle comma separated properties

peterdietz committed:
DS-3027 ItemImport fix to use the collections file collections if present

helix84 committed:
DS-2619 add DSpace version to /rest/status

mwoodiupui committed:
[DS-3024] Exception on attempt to rename a permanent Group

mwoodiupui committed:
[DS-3024] Exception on attempt to rename a permanent Group

helix84 committed:
Merge pull request #1204 from atmire/DS-2851

DS-2851 Administrators cannot create a new user through JSPUI

KevinVdV committed:
[DS-3031] Remove maven warning "Cannot include project artifact ..." in the assembly step

PhilipVis committed:
DS-3030

helix84 committed:
Merge pull request #1270 from atmire/DS-3030

DS-3030 operations with bitstream policies fail in JSPUI

mwoodiupui committed:
[DS-3024] Move permanence logic from entity class to its service class.

mwoodiupui committed:
[DS-3024] Log denied rename attempt.

terrywbrady committed:
remove semicolon from config prop

Commit:
Merge pull request #1269 from KevinVdV/DS-3031-remove-maven-warning

[DS-3031] Remove maven warning "Cannot include project artifact ..." in the assembly step

mwoodiupui committed:
Merge pull request #1044 from mwoodiupui/DS-1837

[DS-1837] Create a separate Maven artifact for the MultiRemoteDSpaceRepositoryHandlePlugin

Note that DSpace/Remote-Handle-Resolver should be released *before* DSpace 6.

helix84 committed:
Merge pull request #1264 from helix84/DS-3016-upgrade-dependencies_building-only

DS-3016 upgrade dependencies before 6.0 release (2)

terrywbrady committed:
auth service allocation

terrywbrady committed:
use config service for props

terrywbrady committed:
handle another array prop

terrywbrady committed:
StringUtils.join for log

tim-atmire committed:
ported xmlui

tdonohue committed:
Merge pull request #1271 from Georgetown-University-Libraries/ds-3032

[DS-3032] Integrate config service properties into ShibAuthentication

tdonohue committed:
Add a testCreateItems(). Decrease number of times we run testCountItems()

mwoodiupui committed:
[DS-3024] Add all database migrations for defining the new EPersonGroup.permanent attribute.

terrywbrady committed:
recreate pr

tdonohue committed:
Enhance unit/integration tests to also test for mapped items in ITCommunityCollection

tdonohue committed:
Fix count of items in a Community by adding new methods to ItemDAO, ItemService. Also cleanup other count methods which are duplicative / badly named.

tdonohue committed:
Remove old CommunityService item count methods, which were inaccurate & slow. ItemService now handles this

tdonohue committed:
Refactor other classes affected by move of item counting from CommunityService to ItemService. Also cleanup horrible spacing/alignment in ItemCounter

tdonohue committed:
Ensure maven-failsafe-plugin uses same memory settings as maven-surefire-plugin

mwoodiupui committed:
Merge branch 'DS-3024' of github.com:mwoodiupui/DSpace into DS-3024

Commit:
Merge pull request #1278 from tdonohue/travis-memory-fixes

Ensure maven-failsafe-plugin uses same memory settings as maven-surefire-plugin

helix84 committed:
DS-3040 fix discovery detection in "dspace version"

KevinVdV committed:
Merge pull request #1276 from terrywbrady/ds-2965d

DS-2965 - Allow cache clear during re-index (V2)

KevinVdV committed:
[DS-3037] Fix completeStep workflow detection

helix84 committed:
DS-2619 removed annotations from setters

helix84 committed:
Merge pull request #1281 from helix84/DS-2619-rest-status-version

DS-2619 add DSpace version to /rest/status

tdonohue committed:
Merge pull request #1247 from KevinVdV/DS-3003-protected-service-dao-constructor

[DS-3003] Ensure all service & DAO implementations have a protected constructor

mwoodiupui committed:
[DS-3024] If the site has renamed the Anonymous and/or Administrator group, rename them back.

helix84 committed:
DS-3040 also report when Discovery is not enabled

mwoodiupui committed:
Merge pull request #1279 from helix84/DS-3040-version-discovery

DS-3040 fix discovery detection in "dspace version"
+1 looks right, simple fix.

mwoodiupui committed:
[DS-3024] Use GroupService.setName, not Group.setName, to implement logic against renaming permanent groups.

mwoodiupui committed:
[DS-2159] Remove XPDF-based filters and configuration

pnbecker committed:
Improves inheritability and logging, removes DOI.setResourceType()

helix84 committed:
DS-3048 change config reload limit from 60 to 5 seconds

cjuergen committed:
Fix for DS-3050

tdonohue committed:
Merge pull request #1285 from helix84/DS-3048-lower-config-reload-check

DS-3048 change config reload limit from 60 to 5 seconds

helix84 committed:
Merge pull request #1286 from cjuergen/DS-3050

DS-3050 XOAI wrong URL encoding

christian-scheible committed:
[DS-2820] fixes XOAI Not filter

helix84 committed:
Merge pull request #919 from tuub/DS-2547

DS-2547: Adds a label to access restricted bitstreams in JSPUI.

tdonohue committed:
DS-3054: Use getArrayProperty() for all multivalued configs. Enhance/correct comments in citation-page.cfg

tdonohue committed:
Correct comment typo

helix84 committed:
Merge pull request #1288 from christian-scheible/DS-2820

[DS-2820] fixes XOAI Not filter

helix84 committed:
fix typos

helix84 committed:
Merge pull request #1290 from tdonohue/DS-3054

DS-3054: PDF Citation Page feature is not reading its configurations correctly

helix84 committed:
Merge pull request #1274 from tim-atmire/full-config-migration

ported xmlui

Commit:
[DS-3056] added and configured xml-maven-plugin to validate all XML and XSL files during the testing phase

Commit:
[DS-3056] suggestion by Andrea Schweer, add .xmap and .xconf extensions to the validation sets

KevinVdV committed:
[DS-3057] Enable enforce versions to prevent dependency issues

KevinVdV committed:
[DS-3046] NPE upon first LDAP login with autoregistration

helix84 committed:
Merge pull request #1293 from KevinVdV/DS-3046-service-init-ldap

[DS-3046] NPE upon first LDAP login with autoregistration

PhilipVis committed:
DS-2887

PhilipVis committed:
DS-2887

PhilipVis committed:
Merge remote-tracking branch 'origin/DS-2887' into DS-2887

Conflicts:
	dspace-xmlui/src/main/java/org/dspace/app/xmlui/cocoon/BitstreamReader.java

tdonohue committed:
Merge pull request #1294 from atmire/DS-2887

DS-2887 AuthorizeException stacktrace shown instead of restricted access page

tdonohue committed:
DS-3058: Fix creating new policies. -1 MUST be passed as the policy ID

bram-atmire committed:
DS-3059 retrieve all supported locales

pnbecker committed:
DS-3061: Use workflow.framework instead of workflow.workflow.framework

tdonohue committed:
Merge pull request #1299 from bram-atmire/master

DS-3059 retrieve all supported locales

tdonohue committed:
Merge pull request #1298 from tdonohue/DS-3058

DS-3058: Fix creating new resource policies. -1 MUST be passed as the policy ID

Commit:
DS-3063 Ensure proper access to news files

mwoodiupui committed:
[DS-3024] Add tests.

tdonohue committed:
Merge pull request #1301 from tuub/DS-3061

DS-3061: Use workflow.framework instead of workflow.workflow.framework

mwoodiupui committed:
Merge pull request #1282 from mwoodiupui/DS-2159

[DS-2159] Deprecate XPDF in DSpace 5, remove in DSpace 6

tdonohue committed:
Merge pull request #1267 from LongsightGroup/DS-3027-item-import-collections

DS-3027 ItemImport fix to use the collections file

mwoodiupui committed:
Merge pull request #1265 from mwoodiupui/DS-3024

[DS-3024] "Administrator" and "Anonymous" groups can be renamed, which would cause them to no longer function in 6.x

Commit:
Merge pull request #1291 from hardyoyo/DS-3056-validate-all-xml-during-testing

[DS-3056] added and configured xml-maven-plugin to validate all XML

terrywbrady committed:
Make String[] of MIME a const per feedback

tdonohue committed:
Merge pull request #1266 from Georgetown-University-Libraries/ds-3026

[DS-3026] REST Reports - Rely on Apache Commons to parse comma separated properties

tdonohue committed:
Merge pull request #1277 from tdonohue/DS-3039

DS-3039 : Fix Community item counting logic. Also cleanup item counting elsewhere in code

tdonohue committed:
Update Flyway migration to keep searching for browse index tables until three are not found

tdonohue committed:
Recreate the old SolrBrowseCreateDAO functionality (necessary for 'metadata' indexes) as a SolrServiceIndexingPlugin

tdonohue committed:
Minor bug fixes to previous commit

helix84 committed:
Merge pull request #1251 from tdonohue/DS-2188-remove-db-browse

DS-2188 : Remove remaining artifacts/configs of DBMS Browse system

bram-atmire committed:
DS-3059 Replacing getProperty with getArrayProperty

KevinVdV committed:
[DS-3062] Fixing issues that prevented the xmlworkflow from being enabled.

tdonohue committed:
Merge pull request #1311 from bram-atmire/DS-3059

DS-3059 Replacing getProperty with getArrayProperty

helix84 committed:
DS-3076 All REST API endpoints fail with Accept: application/json

revert jackson upgrade 2.7.0 -> 2.5.3

http://stackoverflow.com/questions/34721851/spring-4-2-3-and-fasterxml-jackson-2-7-0-are-incompatible

mwoodiupui commented on dspace-api/src/main/java/org/dspace/storage/bitstore/BitstreamStorageManager.java:
```diff
@@ -561,150 +563,155 @@ public static void delete(Context context, int id) throws SQLException
+        // next time around
+        catch (SQLException sqle) {
+            System.err.println("Error: " + sqle.getMessage());
             if (verbose)
```
While you are at it, why not remove the condition test here and always report the exception regardless of 'verbose'?  That would take care of DS-3074 as well.  It is strange that we don't report an event which aborts the command.

helix84 committed:
Merge pull request #1316 from helix84/DS-3076-revert-jackson

DS-3076 All REST API endpoints fail with Accept: application/json

mwoodiupui commented on dspace-api/src/main/java/org/dspace/storage/bitstore/BitstreamStorageManager.java:
```diff
@@ -561,150 +563,155 @@ public static void delete(Context context, int id) throws SQLException
-        }
-        catch (IOException ioe)
-        {
+                System.err.println(sqle);
```
Why do we print the exception twice if 'verbose'?

mwoodiupui commented on dspace-api/src/main/java/org/dspace/storage/bitstore/BitstreamStorageManager.java:
```diff
@@ -561,150 +563,155 @@ public static void delete(Context context, int id) throws SQLException
-            }
-            context.abort();
-            throw ioe;
+                System.err.println(ioe);
```
Why do we print the exception twice if 'verbose'?

tdonohue committed:
Fix API classes using comma-separated configs (and ensure they all use ConfigurationService)

tdonohue committed:
Fix JSPUI classes/jsps that use comma-separated configs (and ensure they all use ConfigurationService)

tdonohue committed:
Fix SWORD classes that use comma-separated configs (and ensure they all use ConfigurationService)

tdonohue committed:
Fix SWORDv2 classes that use comma-separated configs (and ensure they all use ConfigurationService). Correct insecure default value for "on-behalf-of.enable".

tdonohue committed:
Fix XMLUI classes that use comma-separated configs (and ensure they all use ConfigurationService).

pnbecker committed:
Merge pull request #885 from tuub/DS-1349

DS-1349: Add configuration to hide submitter in item history (JSPUI).

KevinVdV committed:
[DS-3057] Enable enforce versions to prevent dependency issues (Fixing dspace-api & dspace-xmlui)

mateuszneumann committed:
DS-3080 Counting withdrawn items (bugfix)

mateuszneumann committed:
DS-3080 Counting withdrawn items (bugfix)

Commit:
DS-3081: Corrected permanent groups Oracle script

tdonohue committed:
Fix more API classes using comma-separated configs (and ensure they all use getArrayProperty)

tdonohue committed:
Fix more JSPUI classes using comma-separated configs (and ensure they all use getArrayProperty)

KevinVdV committed:
[DS-3057] Enable enforce versions to prevent dependency issues (Additional fixing for dspace-api, dspace-xmlui & fixing dspace-rdf, dspace-solr & dspace-sword(v2))

KevinVdV committed:
[DS-3057] Enable enforce versions to prevent dependency issues (Fixing oai)

KevinVdV committed:
Merge pull request #1321 from tomdesair/feature/DS-3081_Correct_permanent_groups_Oracle_script

DS-3081: Corrected permanent groups Oracle script

tdonohue committed:
Merge pull request #1320 from mateuszneumann/master

DS-3080 Counting withdrawn items (bugfix)

KevinVdV committed:
Merge pull request #900 from tuub/DS-2497

DS-2497: Keep version numbers stable.

AmberPoo1 committed:
Correct Sherpa/Romeo ungraded journal (gray) error message

https://jira.duraspace.org/browse/DS-3085

Resolved: DSpace DS-3085 (Ticket in the Space Issue Tracker),
Sherpa/Romeo ungraded journal (gray) shows error

helix84 committed:
DS-3085 fix style in Mirage

helix84 committed:
Merge pull request #1324 from AmberPoo1/fix-sherpa/romeo-gray-submission-error

Correct Sherpa/Romeo ungraded journal (gray) error message

tdonohue committed:
Fix typo in config key

tdonohue committed:
Merge pull request #924 from dbaker3/ds-2318

[DS-2318] Added img-thumbnail class to item-view and instances of 'No Th...

tdonohue committed:
Merge pull request #1202 from KevinVdV/DS-2933-fix-swordclient-xmlui

[DS-2933] Fix SwordClient in xmlui

tdonohue committed:
Merge pull request #1188 from KevinVdV/DS-2914-community-filiator-fix

[DS-2914] Community filiator script fix

tdonohue committed:
Very minor code cleanup (after final review)

tdonohue committed:
Merge pull request #1317 from tdonohue/DS-3075

DS-3075: Comma-separated configurations now need to be loaded via getArrayProperty()

KevinVdV committed:
[DS-3057] Enable enforce versions to prevent dependency issues (Fixing dspace-parent & dspace pom.xml)

KevinVdV committed:
[DS-3057] Enable enforce versions to prevent dependency issues (Fixing jboss logging dependency)

tdonohue committed:
Merge pull request #1313 from KevinVdV/DS-3062-fix-xml-workflow

[DS-3062] Fixing issues that prevented the xmlworkflow from being enabled

peterdietz committed:
Merge pull request #1292 from KevinVdV/DS-3057-enforce-depenceny-versions

[DS-3057] Enable enforce versions to prevent dependency issues

mwoodiupui commented:
Can this be updated in time for 6.0?

mwoodiupui commented:
The documentation is in readme.wiki, apparently to be copied into Confluence when this goes in.

peterdietz committed:
Merge pull request #879 from tuub/DS-2490

DS-2490: adds org.dspace.identifier.VersionedDOIIdentifierProvider

mwoodiupui committed:
Merge pull request #1010 from akinom/DS-2696-showJSP-catch-all-exceptions

DS-2696: JSPUI catch-all-exceptions

Commit:
resolve conflicts - make compatible with BitstreamStorageService

akinom commented:
I fixed the code so it merges successfully
and reset the head of  akinom:prq_cleanup     with   git reset —hard to the merged code

comparing the branches directly  gives a short list of changes
https://github.com/DSpace/DSpace/compare/master...akinom:prq_cleanup?expand=1

But the linked ‘View pull request<https://github.com/DSpace/DSpace/pull/539>’  shows a crazy long list of changes

I have no idea why that happens - so I don’t know how to fix this

Monika


—
Monika Mevenkamp
Digital Repository Infrastructure Developer
Princeton University
Phone: 609-258-4161
Skype: mo-meven

On Mar 16, 2016, at 3:20 PM, Mark H. Wood <notifications@github.com<mailto:notifications@github.com>> wrote:


Can this be updated in time for 6.0?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/DSpace/DSpace/pull/539#issuecomment-197499073>

tdonohue commented:
Closing as this has been replaced by #1337
